etcdserver/etcdserverpb/rpc.proto

```diff
@@ -423,6 +423,10 @@ message PutRequest {
   // If prev_kv is set, etcd gets the previous key-value pair before changing it.
   // The previous key-value pair will be returned in the put response.
   bool prev_kv = 4;
+
+  // If no_value is set, etcd updates the key using its current value.
+  // Returns an error if the key does not exist.
+  bool no_value = 5;
```

Comment:
@heyitsanthony can we keep it as simply lease_update since its doing that only, and will be clear in its meaning?